website/docs/r/launch_template.html.markdown

```diff
@@ -113,6 +113,7 @@ The following arguments are supported:
   `vpc_security_group_ids` instead.
 * `vpc_security_group_ids` - A list of security group IDs to associate with.
 * `tag_specifications` - The tags to apply to the resources during launch. See [Tags](#tags) below for more details.
+* `tags` - (Optional) A mapping of tags to assign to the launch template.
```

Comment:
Nitpick: I'm not sure what the right answer here is (maybe moving this up a line?) but since the line above says `See [Tags](#tags) below for more details.` and this one is right below, that might be confusing at first. 😄